api_docs/include/rest-endpoints.md

```diff
@@ -89,6 +89,8 @@
 * [Remove alert words](/api/remove-alert-words)
 * [Add an APNs device token](/api/add-apns-device-token)
 * [Remove an APNs device token](/api/remove-apns-device-token)
+* [Add an Android GCM registration ID](/api/add-android-reg-id)
+* [Remove an Android GCM registration ID](/api/remove-android-reg-id)
```

Comment:
Let's put all of these in the `Specialty endpoints` section; that's basically where we put client-specific endpoints right now, which these very much are.